test: led:  mimxrt1060_evk@C/mimxrt1062/qspi : build failure

### Describe the bug

platform:  mimxrt1060_evk
git bisect result:
aeb0c0aec18866edb74c89b101dc9f51d616db1b
```
Author: Jeppe Odgaard <jeppe.odgaard@prevas.dk>
Date:   Mon Apr 7 15:31:14 2025 +0200

    tests: drivers: build_all: led: add led_pwm

    Enable test for LED PWM driver.

    Signed-off-by: Jeppe Odgaard <jeppe.odgaard@prevas.dk>

 tests/drivers/build_all/led/app.overlay | 15 +++++++++++++++
 tests/drivers/build_all/led/prj.conf    |  1 +
 2 files changed, 16 insertions(+)

```

### Regression

- [x] This is a regression.

### Steps to reproduce

scripts/twister -p mimxrt1060_evk@C/mimxrt1062/qspi -T tests/drivers/build_all/led --build-only

### Relevant log output

```shell
In file included from /home/shared/disk/zephyr_project/zephyr_test/zephyr/include/zephyr/sys/util_macro.h:34,
                 from /home/shared/disk/zephyr_project/zephyr_test/zephyr/include/zephyr/irq_multilevel.h:16,
                 from /home/shared/disk/zephyr_project/zephyr_test/zephyr/include/zephyr/devicetree.h:20,
                 from /home/shared/disk/zephyr_project/zephyr_test/zephyr/include/zephyr/device.h:12,
                 from /home/shared/disk/zephyr_project/zephyr_test/zephyr/include/zephyr/drivers/led.h:27,
                 from /home/shared/disk/zephyr_project/zephyr_test/zephyr/drivers/led/led_pwm.c:14:
/home/shared/disk/zephyr_project/zephyr_test/zephyr/include/zephyr/device.h:96:41: error: '__device_dts_ord_162' undeclared here (not in a function); did you mean '__device_dts_ord_612'?
   96 | #define DEVICE_NAME_GET(dev_id) _CONCAT(__device_, dev_id)
      |                                         ^~~~~~~~~
/home/shared/disk/zephyr_project/zephyr_test/zephyr/include/zephyr/sys/util_internal.h:72:26: note: in definition of macro '__DEBRACKET'
   72 | #define __DEBRACKET(...) __VA_ARGS__
      |                          ^~~~~~~~~~~
/home/shared/disk/zephyr_project/zephyr_test/zephyr/include/zephyr/sys/util_internal.h:64:9: note: in expansion of macro '__GET_ARG2_DEBRACKET'
   64 |         __GET_ARG2_DEBRACKET(one_or_two_args _if_code, _else_code)
      |         ^~~~~~~~~~~~~~~~~~~~
/home/shared/disk/zephyr_project/zephyr_test/zephyr/include/zephyr/sys/util_internal.h:59:9: note: in expansion of macro '__COND_CODE'
   59 |         __COND_CODE(_XXXX##_flag, _if_1_code, _else_code)
      |         ^~~~~~~~~~~
/home/shared/disk/zephyr_project/zephyr_test/zephyr/include/zephyr/sys/util_macro.h:204:9: note: in expansion of macro 'Z_COND_CODE_1'
  204 |         Z_COND_CODE_1(_flag, _if_1_code, _else_code)
      |         ^~~~~~~~~~~~~
/home/shared/disk/zephyr_project/zephyr_test/zephyr/include/zephyr/devicetree.h:5083:9: note: in expansion of macro 'COND_CODE_1'
 5083 |         COND_CODE_1(DT_HAS_COMPAT_STATUS_OKAY(DT_DRV_COMPAT),   \
      |         ^~~~~~~~~~~
/home/shared/disk/zephyr_project/zephyr_test/zephyr/include/zephyr/toolchain/common.h:169:23: note: in expansion of macro '_DO_CONCAT'
  169 | #define _CONCAT(x, y) _DO_CONCAT(x, y)
      |                       ^~~~~~~~~~
/home/shared/disk/zephyr_project/zephyr_test/zephyr/include/zephyr/device.h:96:33: note: in expansion of macro '_CONCAT'
   96 | #define DEVICE_NAME_GET(dev_id) _CONCAT(__device_, dev_id)
      |                                 ^~~~~~~
/home/shared/disk/zephyr_project/zephyr_test/zephyr/include/zephyr/device.h:297:37: note: in expansion of macro 'DEVICE_NAME_GET'
  297 | #define DEVICE_DT_NAME_GET(node_id) DEVICE_NAME_GET(Z_DEVICE_DT_DEV_ID(node_id))
      |                                     ^~~~~~~~~~~~~~~
/home/shared/disk/zephyr_project/zephyr_test/zephyr/include/zephyr/device.h:314:34: note: in expansion of macro 'DEVICE_DT_NAME_GET'
  314 | #define DEVICE_DT_GET(node_id) (&DEVICE_DT_NAME_GET(node_id))
      |                                  ^~~~~~~~~~~~~~~~~~
/home/shared/disk/zephyr_project/zephyr_test/zephyr/include/zephyr/drivers/pwm.h:260:24: note: in expansion of macro 'DEVICE_DT_GET'
  260 |                 .dev = DEVICE_DT_GET(DT_PWMS_CTLR_BY_IDX(node_id, idx)),       \
      |                        ^~~~~~~~~~~~~
/home/shared/disk/zephyr_project/zephyr_test/zephyr/include/zephyr/drivers/pwm.h:328:34: note: in expansion of macro 'PWM_DT_SPEC_GET_BY_IDX'
  328 | #define PWM_DT_SPEC_GET(node_id) PWM_DT_SPEC_GET_BY_IDX(node_id, 0)
      |                                  ^~~~~~~~~~~~~~~~~~~~~~
/home/shared/disk/zephyr_project/zephyr_test/zephyr/twister-out/mimxrt1060_evk@C_mimxrt1062_qspi/zephyr/tests/drivers/build_all/led/drivers.led.build/zephyr/include/generated/zephyr/devicetree_generated.h:21078:51: note: in expansion of macro 'PWM_DT_SPEC_GET'
21078 | #define DT_N_S_pwmleds_FOREACH_CHILD_SEP(fn, sep) fn(DT_N_S_pwmleds_S_green_pwm_led)
      |                                                   ^~
/home/shared/disk/zephyr_project/zephyr_test/zephyr/include/zephyr/devicetree.h:5367:24: note: in expansion of macro 'DT_N_S_pwmleds_FOREACH_CHILD_SEP'
 5367 | #define DT_CAT(a1, a2) a1 ## a2
      |                        ^~
/home/shared/disk/zephyr_project/zephyr_test/zephyr/include/zephyr/devicetree.h:4015:9: note: in expansion of macro 'DT_FOREACH_CHILD_SEP'
 4015 |         DT_FOREACH_CHILD_SEP(DT_DRV_INST(inst), fn, sep)
      |         ^~~~~~~~~~~~~~~~~~~~
/home/shared/disk/zephyr_project/zephyr_test/zephyr/drivers/led/led_pwm.c:125:9: note: in expansion of macro 'DT_INST_FOREACH_CHILD_SEP'
  125 |         DT_INST_FOREACH_CHILD_SEP(id, PWM_DT_SPEC_GET, (,))     \
      |         ^~~~~~~~~~~~~~~~~~~~~~~~~
/home/shared/disk/zephyr_project/zephyr_test/zephyr/twister-out/mimxrt1060_evk@C_mimxrt1062_qspi/zephyr/tests/drivers/build_all/led/drivers.led.build/zephyr/include/generated/zephyr/devicetree_generated.h:88190:43: note: in expansion of macro 'LED_PWM_DEVICE'
88190 | #define DT_FOREACH_OKAY_INST_pwm_leds(fn) fn(0) fn(1)
      |                                           ^~
/home/shared/disk/zephyr_project/zephyr_test/zephyr/include/zephyr/sys/util_internal.h:105:36: note: in expansion of macro 'DT_FOREACH_OKAY_INST_pwm_leds'
  105 | #define UTIL_PRIMITIVE_CAT(a, ...) a##__VA_ARGS__
      |                                    ^
/home/shared/disk/zephyr_project/zephyr_test/zephyr/drivers/led/led_pwm.c:140:1: note: in expansion of macro 'DT_INST_FOREACH_STATUS_OKAY'
  140 | DT_INST_FOREACH_STATUS_OKAY(LED_PWM_DEVICE)
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~
[180/204] Linking C static library zephyr/kernel/libkernel.a
ninja: build stopped: subcommand failed.
```

### Impact

Annoyance – Minor irritation; no significant impact on usability or functionality.

### Environment

- OS: ubuntu
- toolchain: v0.17.1
- version: v4.1.0-7044-g3e704256e30

### Additional Context

_No response_

--- a/boards/nxp/mimxrt1060_evk/mimxrt1060_evk_mimxrt1062_qspi_B.overlay
+++ b/boards/nxp/mimxrt1060_evk/mimxrt1060_evk_mimxrt1062_qspi_B.overlay
@@ -4,6 +4,13 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+/ {
+	/* FLEXPWM not routed to LED on this EVK */
+	pwmleds {
+		status = "disabled";
+	};
+};
+
 /* FLEXPWM not routed to LED on this EVK */
 &flexpwm2_pwm3 {
 	status = "disabled";

--- a/boards/nxp/mimxrt1060_evk/mimxrt1060_evk_mimxrt1062_qspi_C.overlay
+++ b/boards/nxp/mimxrt1060_evk/mimxrt1060_evk_mimxrt1062_qspi_C.overlay
@@ -9,6 +9,11 @@
 		i2s-codec-tx = &sai1;
 		i2s-tx = &sai1;
 	};
+
+	/* FLEXPWM not routed to LED on this EVK */
+	pwmleds {
+		status = "disabled";
+	};
 };
 
 /* FLEXPWM not routed to LED on this EVK */

